browser-switch/src/main/java/com/braintreepayments/api/ChromeCustomTabsInternalClient.java

```diff
@@ -0,0 +1,26 @@
+package com.braintreepayments.api;
+
+import android.content.Context;
+import android.net.Uri;
+
+import androidx.annotation.VisibleForTesting;
+import androidx.browser.customtabs.CustomTabsIntent;
+
+class ChromeCustomTabsInternalClient {
```

Comment:
We added a "wrapper" around Chrome custom tabs to get Robolectric / Mockito to cooperate. We may consider this a best practice in the future for Android since it helps keep the tests clean by isolating our code from 3rd-party code.